src/app/donation-complete/donation-complete.component.ts

```diff
@@ -65,10 +65,7 @@ export class DonationCompleteComponent implements OnInit {
     private route: ActivatedRoute,
     private sanitizer: DomSanitizer,
   ) {
-    route.params.pipe().subscribe(params => {
-      this.donationId = params.donationId;
-      this.checkDonation();
-    });
+    this.checkDonation();
```

Comment:
nice simplification - and did the old version have a memory leak because it didn't unsubscribe to the params on destroy?

Reply:
Maybe!